templates/layout.html

```diff
@@ -4,9 +4,12 @@
 <head>
   <!-- Required meta tags -->
   <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-
-  <title>My App</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Slab&display=swap" rel="stylesheet">
+  <link href="{{ url_for('static', filename='stylesheets/main.scss.css') }}" rel="stylesheet" type="text/css">
+  <title>{{ page_name }} - Financial Dashboard</title>
```

Comment:
```suggestion
  <title>{{ page_name }} - Dollar Tracker</title>
```

Should we use our application name instead of "Financial Dashboard"?